app/build.gradle

```diff
@@ -65,6 +83,10 @@ android {
         buildConfigField "String", "NPS_SURVEY_URL", '"https://dev-wallet-net-promoter-score.radixdlt.com/"'
         buildConfigField "String", "REFINER_FORM_UUID", '"281622a0-dc6b-11ee-8fd1-23c96056fbd2"'
         resValue "string", "app_name", "Radix Wallet"
+
+        def properties = secretProperties()
+        buildConfigField "String", "APPS_FLYER_DEV_KEY", buildConfigProperty(properties, 'appsFlyerDevKey')
```

Comment:
will there be different key for release build?

Reply:
so maybe we should remove `_DEV_` in this scenario from key name?

Reply:
I'll change the name to avoid confusion, but FYI dev in this case is not related to the environment, it's actually what the key is called on AF platform.